cime_config/machines/config_batch.xml

```diff
@@ -56,6 +56,14 @@
     </directives>
   </batch_system>
 
+   <batch_system MACH="spock" type="slurm">
+     <queues>
+       <queue walltimemax="03:00:00" default="true">ecp</queue>
+       <queue walltimemax="03:00:00">caar</queue>
+       <queue walltimemax="12:00:00">batch</queue>
```

Comment:
Please add in min and max node-counts for the queues:
```
       <queue strict="true" nodemin="1" nodemax="4" walltimemax="03:00:00" default="true">ecp</queue>
       <queue strict="true" nodemin="1" nodemax="4" walltimemax="03:00:00">caar</queue>
       <queue strict="true" nodemin="5" nodemax="16" walltimemax="01:00:00">caar</queue>
```
I couldn't submit to `batch` queue and it's not listed in the docs.
```
srun: error: You are not a member of partition batch. Please specify a valid partition with the -p flag. Valid partitions are:
srun: error:     ecp
```